### PR TITLE
HTML&CSS: Newsweek Clone project

### PR DESCRIPTION
**Pull request by brugobi**
## Newsweek Clone project 

- the most import on this project is using bootstrap;
- It is responsive;

Thank you for the reviewer

### CODE REVIEW

**Comment by serdg0** [comment]
Hello @brugobi,
I had to mark this submission as invalid, because it doesn't show the lintering in the PR.

Feel free to message me in slack if anything comes up!.

**Comment by brugobi** [comment]
Hello Sérgio, sorry about that I still learning
Captura de Tela 2020-05-09 às 07 36 56
 , I forget to enable but now is ON

**Comment by brugobi** [comment]
Hello Sérgio, sorry about that, I forget to enable but now is

**Comment by lucasmazo32** [comment]
# Project under review 🔭

File: assets/stylesheet/index.css [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+  display: none;
+}
+
+.cuture-travel-section-mobile {
+  display: none;
+}
+
+.latest-news-mobile {
+  display: none;
+}
```
Comment (lucasmazo32) [inline assets/stylesheet/index.css]: When you have multiple display nones with classes you can change it into maximum three display none. You can have one for each breakpoint and you won't have the need to create multiple classes. Kindly fix that.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1796 @@
...
+                                            </a>
+                                        </p>
+                                    </div>    
+                                    <div class="summary-heros">
+                                        Fans may be lamenting the end of the 'Star Wars' series, but they can still visit these film locations and space-themed destinations.
+                                    </div>
+                                </div>
+                            </div>        
+                        </div>
+                        <div class="col-lg opinion latest-news-mobile">
```
Comment (lucasmazo32) [inline index.html]: Kindly add the Signup for more newsletter field.

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1796 @@
...
+                                <h6 class="mt-0 mb-1"><a href="/">Glowing Blue Waves Reported Across Southern California Due to 'Red Tide'</a></h6>
+                              </div>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="row sports"> 
+                    <div class="col-lg-4 col-md-6">
+                        <div class="block-title"><a class="text-danger font-weight-bold" style="text-decoration: none;" href="/">CULTURE</a></div>
```
Comment (lucasmazo32) [inline index.html]: Kindly change this title to capitalize (Culture) instead of uppercase.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1796 @@
...
+                                <h6 class="mt-0 mb-1"><a href="/">These Celebrities Embraced The Met Gala Regarless Of Covid-19</a></h6>
+                              </div>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                
+                <div class="row sports"> 
+                    <div class="col-lg-4 col-md-6">
+                        <div class="block-title"><a class="text-danger font-weight-bold" style="text-decoration: none;" href="/">SPORTS</a></div>
```
Comment (lucasmazo32) [inline index.html]: Kindly change this title to capitalize instead of uppercase.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1796 @@
...
+                    <div class="col-lg-4 col-md-6">
+                        <div class="invisible block-title font-weight-bold text-muted">U.S.</div>
+                        <div class="card">
+                            <img class="card-img-top" src="https://d.newsweek.com/en/full/1587503/pennsylvania-governor-tom-wolf.jpg?w=397&amp;h=265&amp;f=a901a37a56354bbff56704d6f47e5a0a" alt="...">
+                        </div>
+                        <div class="categorie">
+                          <h3 class="feature-title">
+                            <a href="/">
+                                Pennsylvania Gym Reopens Despite Lockdown Order
+                            </a>
```
Comment (lucasmazo32) [inline index.html]: This section on the Newsweek page stays the same in desktop and tablet. I will add the screenshot from tablet.

> Original
image

> Your page
image

Kindly fix that

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1796 @@
...
+                        </ul>
+                    </div>
+                </div>
+                
+                <div class="row magazine-section">
+                    <article class="col-lg-7 col-md-12 col-sm-12">
+                        <div class="text-danger font-weight-bold">SUBSCRIBE</div>
+                        <div class="row no-gutters magazine">
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <img src="https://d.newsweek.com/en/full/1585087/may-08-15-2020-cover.jpg?w=170&amp;q=95&amp;f=c4d10af6474d34e586607b280b6589f0" class="card-img" alt="...">
```
Comment (lucasmazo32) [inline index.html]: This section uses two images in the original webpage. Kindly fix that. I will add a screenshot from the original page.

image

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1796 @@
...
+                        <div class="text-danger font-weight-bold">NEWSLETTER</div>
+                        <div class="row no-gutters">
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <img src="https://d.newsweek.com/en/full/1541302/starting5-homepage.png?w=397&amp;h=265&amp;q=90&amp;f=742bce9a6f72dd8f19ba2e7fd5db06e5" style="height: 200px;" class="card-img" alt="...">
+                            </div>
+                            <div class="col-lg-8 col-md-8 col-sm-12">
+                                <div class="card-body">
+                                    <h5 class="card-title">The Starting 5</h5>
+                                    <p class="card-text text-muted">See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+                                    <a href="#" class="btn btn-danger">SIGN UP&nbsp;&nbsp;></a>
```
Comment (lucasmazo32) [inline index.html:1769]: Please hide the image for the last section in the phone version.

> Your page
image

> Original 
image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,1796 @@
...
+                        <div class="flex-fill">
+                            <span id="logo-footer">Newsweek</span>
+                            <span id="Newsweek-2020">© 2020 NEWSWEEK</span>
+                        </div>
+                        <div class="align-self-center footer-icons">
+                            <i class="fa fa-facebook-square" aria-hidden="true"></i>
+                            <i class="fa fa-twitter" aria-hidden="true"></i>
+                            <i class="fa fa-instagram" aria-hidden="true"></i>
+                            <i class="fa fa-tumblr" aria-hidden="true"></i>
+                            <i class="fa fa-linkedin" aria-hidden="true"></i>
```
Comment (lucasmazo32) [inline index.html]: Each of these icons should be enclosed by links (```<a href="#"><i ... </i></a>```). In the original webpage when you hover them, they change the colors. It's a nice detail that you can copy!

Also, in phone version, they are below the Newsweek logo.

> Original
image

> Your page
image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

Before the projects get approved, there are some changes that you need to make. Mostly they are simple but improve an already great page.

### Happy coding! :computer:

## Lucas Mazo

**Comment by brugobi** [comment]
thank you @lucasmazo32 for your review I really appreciate that and I  Kindly did all the recommendations

**Review by jofer86 — APPROVED** [review]
## :wave: :wave: Hello!, I'm Jorge, and  I'm here to review this Pull Request, let's take a look shall we :eye: :eye: :eye: 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- All of the required changes were taken care off. This project looks great as it is. Great job!!

`status: Approved` :rocket: :rocket: :+1: :+1: 
--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

**Pull request merged**